### PR TITLE
feat: completed wishlist page

### PR DESCRIPTION
**Pull request by Gautam2305**
FEATURES ADDED
- main nav
- card component with 2 CTAs , one for buying directly & another for adding to bag
-  original price with text-decoration: line-through
- discount percentage

Screenshot (123)

### CODE REVIEW

**Review by sadathshariff — COMMENTED** [review]
Everything looks good, just a suggestion to have different colors for add to cart and buy now it will be easy to visualize and for discount increase the font-weight a bit

File: pages/wishlist_page/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    max-width: 100%;
+    position: relative;
+}
+.wishlist-btn{
+    position: absolute;
+    top: 0.5rem;
+    right: 0.5rem;
+    color: var(--st-mid-red);
+    background-color: var(--st-white);
+    padding: 5px 0.5rem;
```
Comment (ttahm3d) [inline pages/wishlist_page/wishlist.css]: Dont use different units for same property.

File: pages/wishlist_page/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="/assets/s-solid.svg" />
+    <link rel="stylesheet" href="/style.css">
+    <script src="https://kit.fontawesome.com/184c79d39f.js" crossorigin="anonymous"></script>
+    <title> Sierra </title>
+</head>
+<body>
+    <nav class="main-nav">
+        <div class="nav-category">
+            <h3 class="brand-name">S I E R R A</h3>
```
Comment (ttahm3d) [inline pages/wishlist_page/wishlist.html]: Dont manually add the space in html markup. Change it through letter-spacing

```css
landing-spacing: 4px;
```

**Review by ttahm3d — COMMENTED** [review]
Added comments please take a look

File: pages/wishlist_page/wishlist.css (hunk 1 above)
Comment (Gautam2305) [reply pages/wishlist_page/wishlist.css]: okay

File: pages/wishlist_page/wishlist.html (hunk 2 above)
Comment (Gautam2305) [reply pages/wishlist_page/wishlist.html]: okayyy

Comment (Gautam2305) [reply pages/wishlist_page/wishlist.html]: done

File: pages/wishlist_page/wishlist.css (hunk 1 above)
Comment (Gautam2305) [reply pages/wishlist_page/wishlist.css]: done

**Comment by Gautam2305** [comment]
> Everything looks good, just a suggestion to have different colors for add to cart and buy now it will be easy to visualize and for discount increase the font-weight a bit

did the required changes

**Review by ttahm3d — APPROVED** [review]

**Review by sadathshariff — APPROVED** [review]

**Pull request merged**